src/ziggurat/producer.clj

```diff
@@ -121,3 +126,10 @@
      (let [error-msg (str "Can't publish data. No producers defined for stream config [" stream-config-key "]")]
        (log/error error-msg)
        (throw (ex-info error-msg {:stream-config-key stream-config-key}))))))
+
+(defn -send
+  ([stream-config-key topic key value]
```

Comment:
Are we expecting keyword here or string?

Reply:
string, all arguments passed to java exposed functions are native to java.

Reply:
Then why stream config `key` ?

Reply:
because it is the key to the producers' map, the `key` here refers to the key of the map and not that it is a keyword